### PR TITLE
Replace vmandnot and vmornot with vmandn and vmorn - Issue #421

### PR DESCRIPTION
**Pull request by ShivangMishra**

### CODE REVIEW

**Review by Alasdair — COMMENTED** [review]
Wondering if we should also change `MM_VMANDNOT` to `MM_VMANDN`? Otherwise lgtm.

**Comment by ShivangMishra** [comment]
> Wondering if we should also change `MM_VMANDNOT` to `MM_VMANDN`? Otherwise lgtm.



> Wondering if we should also change `MM_VMANDNOT` to `MM_VMANDN`? Otherwise lgtm.

so...should I change it?

**Comment by ShivangMishra** [comment]
looks ok now?

**Review by Alasdair — COMMENTED** [review]
I think the indention needs adjustment, but otherwise should be fine.

File: model/riscv_insts_vext_mask.sail [hunk 1]
```diff
@@ -51,11 +51,11 @@ function clause execute(MMTYPE(funct6, vs2, vs1, vd)) = {
       result[i] = match funct6 {
         MM_VMAND     => vs2_val[i] & vs1_val[i],
         MM_VMNAND    => not(vs2_val[i] & vs1_val[i]),
-        MM_VMANDNOT  => vs2_val[i] & not(vs1_val[i]),
+        MM_VMANDN  => vs2_val[i] & not(vs1_val[i]),
```
[the remark this answers is not in the document]
Comment (ShivangMishra) [reply model/riscv_insts_vext_mask.sail]: Done

**Review by Alasdair — APPROVED** [review]

**Comment by ShivangMishra** [comment]
@Timmmm @Alasdair can this be merged now?

**Review by Timmmm — APPROVED** [review]
Yeah LGTM. Can you squash your commits?

**Comment by Alasdair** [comment]
You can squash the commits when doing a merge as a maintainer, and it lets you update the message, so I just went ahead and did it.

Thanks for the fix @ShivangMishra

**Pull request merged**